gravitee-apim-portal-webui-next/src/services/redoc.service.ts

```diff
@@ -17,7 +17,6 @@ import { Injectable } from '@angular/core';
 
 import { readYaml } from '../app/helpers/yaml-parser';
 
-// eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
I think you should keep it or find the right class to replace 'any'

Reply:
Done